containers/ecr-viewer/src/styles/custom-styles.scss

```
@@ -184,39 +184,47 @@ h4 {
 
 .sort-button{
   border: none;
-  outline: none;
   transition: none;
+  margin-left: auto;
+  background-color: transparent;
   &:hover{
-    outline: none;
     background-color: transparent; /* Removes the hover background */
     color: inherit; /* Keeps the text color unchanged */
   }
   &:active {
-    outline: none;
-    background-color: inherit; /* Keeps the original background color */
+    background-color: transparent; /* Keeps the original background color */
     color: inherit; /* Keeps the text color unchanged */
   }
   &:focus {
-    outline: none;
-    background-color: inherit; /* Removes fill after click */
+    background-color: transparent; /* Removes fill after click */
+    color: inherit; /* Keeps the text color unchanged */
   }
 }
 
-.sortable-column{
-  background-color: transparent;
-  @include add-background-svg("usa-icons/sort_arrow");
+.sortable-column, .sortable-desc-column, .sortable-asc-column{
+    &.usa-button{
+      margin-left: auto;
+      justify-content: flex-end;
+      padding: 24px;
```

```suggestion
      padding: 12px;
```
1/2 image height/width